riva/proto/riva_asr.proto

```diff
@@ -111,6 +111,17 @@ message StreamingRecognizeRequest {
   RequestId id = 100;
 }
 
+/*
+ * EndpointingConfig is used for configure all end of utterance related values
+ */
+message EndpointingConfig {
+	optional int32 start_history = 1;
+	optional float start_threshold = 2;
+	optional int32 stop_history = 3;
+	optional int32 stop_history_eou = 4;
+	optional float stop_threshold = 5;
```

Comment:
let's use 2 spaces of indentation here instead of tab

Reply:
Updated in latest commit. Please check